Adds new specutils data loader for SDSS MaNGA cube/rss data (#702)
astropy/specutils: docs/identify.rst

==============================
Identifying Spectrum1D Formats
==============================

``specutils`` provides a convenience function,
`~specutils.io.registers.identify_spectrum_format`, which attempts to guess the
`~specutils.Spectrum1D` file format from the list of registered formats, and
essentially acts as a wrapper on `~astropy.io.registry.identify_format`.

This function is useful for identifying a spectrum file format without the need to
read it in with Spectrum1d's `~specutils.Spectrum1D.read` method.  It returns the best guess as to a

> valid format from the list of ``Formats`` as given by `~astropy.io.registry.get_formats`.

Reviewer: Perhaps this could spell out still more clearly that it is using the same identification methods as `Spectrum1D.read` – I doubt many users will be familiar with the internals of `~astropy.io.registry`.

Author: I've updated the docs here with another sentence or two.  Let me know what you think.

General usage is as follows, passing in an absolute filename path:

.. code-block:: python

    >>> from specutils.io.registers import identify_spectrum_format
    >>> identify_spectrum_format("/path/to/file.fits")  # doctest: +SKIP

An example is given to identify a SDSS MaNGA data cube file:

.. code-block:: python

    >>> from astropy.utils.data import download_file
    >>> from specutils.io.registers import identify_spectrum_format
    >>>
    >>> url='https://dr15.sdss.org/sas/dr15/manga/spectro/redux/v2_4_3/8485/stack/manga-8485-1901-LOGCUBE.fits.gz'

>     >>> dd = download_file(url)

Reviewer: ```suggestion
    >>> dd = download_file(url)  # doctest: +SKIP
```
This made the doctest fail when not run with `remote_data=True` – don't know if one could add the `remote_data` decorator somehow for doctest, but I'd suggest to simply skip it.

==============================
Identifying Spectrum1D Formats
==============================

``specutils`` provides a convenience function,
`~specutils.io.registers.identify_spectrum_format`, which attempts to guess the
`~specutils.Spectrum1D` file format from the list of registered formats, and
essentially acts as a wrapper on `~astropy.io.registry.identify_format`.

> This function is useful for identifying a spectrum file format without the need to
> read it in with Spectrum1d's `~specutils.Spectrum1D.read` method.  It uses the

Reviewer 2: ```suggestion
This function is useful for identifying a spectrum file format without reading the
whole file with the  `~specutils.Spectrum1D.read` method.  It uses the
```

> same identification method as ``read`` however, so it simply provides a convenience

Reviewer 2: ```suggestion
same identification method as ``read`` however, so it provides a convenience
```

of access outside of calling ``read`` without any change in underlying functionality.
It returns the best guess as to a valid format from the list of ``Formats``
as given by `~astropy.io.registry.get_formats`.

> General usage is as follows, passing in an absolute filename path:
> 
> .. code-block:: python
> 
>     >>> from specutils.io.registers import identify_spectrum_format
>     >>> identify_spectrum_format("/path/to/file.fits")  # doctest: +SKIP
> 
> An example is given to identify a SDSS MaNGA data cube file:
> 

Reviewer 2: I think the "general usage" isn't needed given that the MaNGA example is already quite clear
```suggestion
For eample, to identify a SDSS MaNGA data cube file:

```
